Gemfile.lock

```diff
@@ -52,6 +52,3 @@ DEPENDENCIES
   cucumber (~> 1.3.9)
   ronn
   sinatra
-
-BUNDLED WITH
```

Comment:
sorry that what I get for doing `git commit -am' commit message'` without doing `git status` and `git dif` first!